src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/DataSetFilesControllerTests.cs

```diff
@@ -1874,13 +1899,15 @@ public async Task FetchDataSetDetails_Success()
             viewModel.Meta.GeographicLevels
                 .AssertDeepEqualTo(dataSetFileMeta!.GeographicLevels);
```

Comment:
Can just use `Assert.Equal` rather than deep equals.

Same thing for the other meta facets below as well.